programs/clearing_house/src/math/fees.rs

```diff
@@ -36,7 +37,13 @@ pub fn calculate_fee_for_order_fulfill_against_amm(
         let filler_reward: u128 = if !reward_filler {
             0
         } else {
-            calculate_filler_reward(fee, order_ts, now, &fee_structure.filler_reward_structure)?
+            calculate_filler_reward(
```

Comment:
can you add a test that uses the multiplier?

Reply:
only apply the filler multiplier to matched orders